core/src/main/java/io/methvin/watcher/DirectoryWatcher.java

```diff
@@ -59,15 +59,24 @@
 
   // this is set to true/false depending on whether recursive watching is supported natively
   private Boolean fileTreeSupported = null;
+  private Boolean enableFileHashing = true;
```

Comment:
This doesn't need to be a `Boolean`. It can just be a regular `boolean`. The other one is `Boolean` since I needed to handle the uninitialized state.